meta-mender-nxp/templates/local.conf.append

```diff
@@ -33,6 +34,14 @@ MENDER_STORAGE_DEVICE_imx7s-warp = "/dev/mmcblk1"
 MENDER_UBOOT_ENV_STORAGE_DEVICE_OFFSET_1_imx7s-warp = "0x80000"
 MENDER_UBOOT_ENV_STORAGE_DEVICE_OFFSET_2_imx7s-warp = "0xA0000"
 
+# Specific Configuration for i.MX7D Sabre
+IMAGE_BOOT_FILES_imx7dsabresd = "boot.scr"
+MENDER_UBOOT_STORAGE_INTERFACE_imx7dsabresd = "mmc"
+MENDER_UBOOT_STORAGE_DEVICE_imx7dsabresd = "0"
+MENDER_STORAGE_DEVICE_imx7dsabresd = "/dev/mmcblk0"
```

Comment:
You can also drop these 3 lines in this 
context (`/dev/mmcblk0` is the default value for `MENDER_STORAGE_DEVICE`)

Reply:
Thanks, i remove this